### DISCUSSION TITLE
Does the file source in `consult-buffer` require some specific package to be loaded?

### DISCUSSION THREAD

**Discussion by NightMachinery** [minad/consult#955] [Q&A] [answered]
Whenever I restart emacs, the file source does not appear in `consult-buffer` until I do some actions (e.g., open a file?). I think this is because some package is lazy loaded when I perform that action. I do load `recentf` on the startup though ...

**Comment by oantolin** [comment, marked as answer]
The file source requires, not only that `recentf` be loaded, but also that `recentf-mode` be turned on (its `:enabled`  property is set to  `(lambda () recentf-mode)`). Maybe you are loading `recentf` on startup but only turning it on later, after some condiiton is met?

**Reply by NightMachinery** [reply]
Thanks, this was it.
